firmware/integration/rusefi_config.txt

```diff
@@ -903,7 +903,13 @@ float[CRANKING_ADVANCE_CURVE_SIZE] crankingAdvance    ;Optional timing advance t
 	int16_t coastingFuelCutTps;percent between 0 and 100;"%",        1,     0,  0,    100,  0
 	int16_t coastingFuelCutClt;Fuel cutoff is deactivated if CLT<threshold;"C",        1,     0,  -100,    100,  0
 	
-	int[774] unusedEnd;
+	int16_t pidExtraForLowRpm;+Increases PID reaction for RPM<target by adding extra percent to PID-error;"%",        1,     0,  0,    100,  0
+	int16_t unusedInt16;
+	
+float[CLT_CURVE_SIZE] iacCoastingBins;CLT-based idle position for coasting (used in Auto-PID Idle mode);"C",        1,     0,   -100.0,    250.0,  2
+float[CLT_CURVE_SIZE] iacCoasting;    CLT-based idle position for coasting (used in Auto-PID Idle mode);"%",      1,     0,      0.0,    100.0,   2
+	
```

Comment:
We add a new table for Auto-Idle PID regulator instead of 'Manual' warmup multiplier.
But this time we set IAC positions (%) directly, not just multipliers. In the future, we might want to fill-in this table automatically & store real working values.
Also, there's a good reason to set a little oversized values in this table (i.e. greater than your normal 'Idle' values) just for safety...